packages/polymath-ui/src/components/STOStatus/STOStatus.js

```diff
@@ -133,23 +137,47 @@ export default class STOStatus extends Component<Props> {
               </div>
             </div>
           </div>
-          {countdownProps != null && (
+          {isSaleComplete ? (
             <div className="pui-countdown-container">
-              <Countdown
-                deadline={countdownProps.deadline}
-                title={countdownProps.title}
-                buttonTitle={
-                  notPausable
-                    ? undefined
-                    : this.props.isStoPaused
-                    ? 'RESUME STO'
-                    : 'PAUSE STO'
-                }
-                handleButtonClick={this.props.toggleStoPause}
-                isPaused={this.props.isStoPaused}
-                pausable={!notPausable}
-              />
+              <ContentBox
+                style={{
+                  borderRadius: '10px',
+                  borderTop: '15px solid #00AA5E',
+                  width: '350px',
+                }}
+              >
+                <p align="center">
+                  <Icon
+                    name="checkmark--outline"
+                    fill="#00AA5E"
+                    width="64"
+                    height="64"
+                  />
+                </p>
+                <h2 className="pui-h2" align="center">
```

Comment:
You should use the new components for this.
`<Heading variant="h2" textAlign="center" />`
https://polymath-ui.netlify.com/src-components-heading-heading